src/sqlacodegen/generators.py

```diff
@@ -547,7 +558,7 @@ def render_column_type(self, coltype: object) -> str:
                 if (value := getattr(coltype, colname)) is not None:
                     kwargs[colname] = repr(value)
 
-        if isinstance(coltype, JSONB):
+        if isinstance(coltype, (JSONB, JSON)):
```

Comment:
Does the added test cover this change?

Reply:
No I can parameterize existing test as well. Would you like me to do that?

Reply:
That would be great!

Reply:
Done via a separate test